app/components/admin/schools/relationship_component.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Admin
+  module Schools
+    class RelationshipComponent < ViewComponent::Base
+      attr_accessor :relationship, :school, :school_cohort, :participants, :superuser
+
+      def initialize(relationship:, school:, school_cohort:, superuser:, participants: nil)
+        @relationship = relationship
+        @school = school
+        @school_cohort = school_cohort
+        @participants = participants || relationship.induction_programmes.flat_map(&:participant_profiles)
+        @superuser = superuser
+      end
+
+      def allow_challenge?
+        superuser
+      end
+
+      def actions
+        [
+          if allow_challenge?
+            govuk_link_to(
+              "Challenge relationship",
```

Comment:
Think we need some visually-hidden text here so that it reads as "Challenge relationship [with provider_name]" - otherwise there might be multiple links with the same text on the page. It might even need to include the cohort year if there could be multiple relationships with the same provider?

Reply:
I missed this first time around, will add it now

Reply:
Fixed in f578a74d75d